### PR TITLE
Updated link in README

### LINKED ISSUE #6793: Outdated plugin pylint-i18n linked
On the [pylint pypi page](https://pypi.org/project/pylint/) and the README the plugin [pylint-i18n](https://github.com/amandasaurus/python-pylint-i18n) is linked, which latest commit is from 2014, so 8 years ago.
Since the plugin seems to be outdated, I would suggest to remove the link. What do you think?

### PR DESCRIPTION
**Pull request by PrajwalBorkar**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
|    | :bug: Bug fix          |
|    | :sparkles: New feature |
|   | :hammer: Refactoring   |
| ✓   | :scroll: Docs          |

## Description
Updated link to new repository

Refs #XXXX

Closes #6793

### CODE REVIEW

**Review by Pierre-Sassoulas — APPROVED** [review]
Thank you for this :thumbsup: If you want to fix the other link here too, it's ok for me.

**Comment by PrajwalBorkar** [comment]
> Thank you for this 👍 If you want to fix the other link here too, it's ok for me.

Yes, I will. Which one can u tell?

**Comment by Pierre-Sassoulas** [comment]
> Which one can u tell?

I'm talking about [this comment](https://github.com/PyCQA/pylint/issues/6793#issuecomment-1145123243)

**Comment by PrajwalBorkar** [comment]
> > Which one can u tell?
> 
> I'm talking about [this comment](https://github.com/PyCQA/pylint/issues/6793#issuecomment-1145123243)

Done. Thanks 👍

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 Please read the
-[contribute doc](https://github.com/PyCQA/pylint/blob/main/doc/development_guide/contribute.rst).
+[contribute doc](https://github.com/PyCQA/pylint/blob/main/doc/development_guide/contributor_guide/contribute.rst).
```
Comment (Pierre-Sassoulas) [inline .github/CONTRIBUTING.md:2]:
```suggestion
[contribute doc](https://pylint.pycqa.org/en/latest/development_guide/contributor_guide/contribute.html).
```

**Comment by Pierre-Sassoulas** [comment]
Nice catch for the dead link ! We have a check for that but we missed it because we were linking to the file in the repository and not the actual doc.

**Comment by PrajwalBorkar** [comment]
> Nice catch for the dead link ! We have a check for that but we missed it because we were linking to the file in the repository and not the actual doc.

Yeah actually I was looking for contribution doc then I found that.

**Comment by Pierre-Sassoulas** [comment]
Congratulation on becoming a pylint contributor 😉

**Comment by PrajwalBorkar** [comment]
> Congratulation on becoming a pylint contributor 😉

Thanks :) Maybe it's time to learn python to contribute more ;)

**Comment by Pierre-Sassoulas** [comment]
If you want to start slow we have a lot of messages, and most of them needs to be documented in https://github.com/PyCQA/pylint/issues/5953 😉

**Pull request merged**